src/redturtle/importer/volto/transmogrifier/deserializers/blocks.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+from collective.volto.blocksfield.interfaces import IBlocksField
+from design.plone.contenttypes.upgrades.draftjs_converter import to_draftjs
```

Comment:
Questa è una dipendenza forte con design.plone.contenttypes che in questo prodotto non abbiamo. Riporterei quel metodo qui al massimo.